systemvm/agent/scripts/ssvm-check.sh

```diff
@@ -118,7 +118,7 @@ fi
 # check for connectivity to the management server
 echo ================================================
 echo Management server is $MGMTSERVER.  Checking connectivity.
-socatout=$(echo | socat - TCP:$MGMTSERVER:8250,connect-timeout=3 2>&1)
+socatout=$(echo | socat - TCP:$MGMTSERVER:8250,connect-timeout=3 | tr -d '\0' 2>&1)
```

Comment:
@Pearl1594 what was the issue during the reboot test, does this fix that? If so, do we need to port it back to 4.14/4.13?

Reply:
while it reports a failure in ssvm reboot test, the logs report a failure in ssvm check:
```
2020-08-11 23:39:12,513 - CRITICAL - FAILED: test_07_reboot_ssvm: ['Traceback (most recent call last):\n', '  File "/usr/lib64/python2.7/unittest/case.py", line 369, in run\n    testMethod()\n', '  File "/marvin/tests/smoke/test_ssvm.py", line 897, in test_07_reboot_ssvm\n    self.test_03_ssvm_internals()\n', '  File "/marvin/tests/smoke/test_ssvm.py", line 460, in test_03_ssvm_internals\n    "Check for Errors in tests"\n', '  File "/usr/lib64/python2.7/unittest/case.py", line 553, in assertEqual\n    assertion_func(first, second, msg=msg)\n', '  File "/usr/lib64/python2.7/unittest/case.py", line 546, in _baseAssertEqual\n    raise self.failureException(msg)\n', 'AssertionError: Check for Errors in tests\n']
```